.github/workflows/coverage-check.yaml

```diff
@@ -123,7 +123,6 @@ jobs:
           sudo locale-gen ru_RU
           sudo locale-gen ru_RU.UTF-8
           sudo update-locale
-          ln -s ../data data
```

Comment:
You forgot about linux-check.yaml

Reply:
Good catch! Modified but didn't save the file 😄